### PR TITLE
feat: add documentation for deeplinks with query params

### PR DESCRIPTION
**Pull request by Camvillard**
Added a section to formalize the deeplink query params that should be supported.

### CODE REVIEW

File: reference.md [hunk 1]
```diff
@@ -927,3 +927,23 @@ Field Name | Presence | Type | Description
...
+### deeplink with query params
+
+The uris provided in `realtime_booking` (`android_uri`, `ios_uri`, `web_uri`) and `booking_rules` (`booking_url`) should support a specific set of query params:
+
+Param | Presence | Type | Description
+---|---|---|---
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
+`drop_off_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
+`drop_off_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
```
Comment (gcamp) [inline reference.md:941]:
```suggestion
`dropoff_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
`dropoff_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
```

File: reference.md [hunk 2]
```diff
@@ -927,3 +927,23 @@ Field Name | Presence | Type | Description
...
+`pickup_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be picked-up.
+`drop_off_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be dropped-off.
+
+#### Example
+
+```
+https://www.example.com?pickup_lat=45.52585187926036&pickup_lon=-73.59502716927499&drop_off_lat=45.53614412053443&drop_off_lon=-73.5130921428607&pick_up_time=1726173300000&drop_off_time=1726173600000
```
Comment (gcamp) [inline reference.md:948]:
```suggestion
https://www.example.com?pickup_lat=45.52585187926036&pickup_lon=-73.59502716927499&dropoff_lat=45.53614412053443&dropoff_lon=-73.5130921428607&pickup_time=1726173300000&dropoff_time=1726173600000
```

File: reference.md [hunk 3]
```diff
@@ -927,3 +927,23 @@ Field Name | Presence | Type | Description
...
+The uris provided in `realtime_booking` (`android_uri`, `ios_uri`, `web_uri`) and `booking_rules` (`booking_url`) should support a specific set of query params:
+
+Param | Presence | Type | Description
+---|---|---|---
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
+`drop_off_lat` | REQUIRED | Latitude | Latitude of the location where the user will be dropped off.
+`drop_off_lon` | REQUIRED | Longitude | Longitude of the location where the user will be dropped off.
+`pickup_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be picked-up.
+`drop_off_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be dropped-off.
```
Comment (gcamp) [inline reference.md:943]:
```suggestion
`dropoff_time` | OPTIONAL | Timestamp | Timestamp, in milliseconds, of the moment the user will be dropped-off.
```

Comment (gcamp) [inline reference.md:943]: The Timestamp that we had before this was epoch in seconds. Also we have a Timestamp type with no definition (that was before this PR). Can we fix 🙏

File: reference.md (hunk 1 above)
Comment (Camvillard) [reply reference.md:941]: Shouldn't we keep the same names everywhere? We use `drop_off` for the `wait_time` and `booking_rules` endpoint.

Comment (agregoire) [reply reference.md:941]: Agreed

File: reference.md (hunk 3 above)
Comment (Camvillard) [reply reference.md:943]: Are you refering to the `"last_updated": 1609866247` property?

Comment (gcamp) [reply reference.md:943]: Yes exactly

File: reference.md (hunk 1 above)
Comment (agregoire) [reply reference.md:941]: The "real" spelling (as much as a word spelling can be real) is "drop-off": https://www.merriam-webster.com/dictionary/drop-off.

Comment (gcamp) [reply reference.md:941]: I stand corrected, not sure why I thought it was dropoff. It's the same thing in GTFS.

File: reference.md (hunk 3 above)
Comment (gcamp) [reply reference.md]: Sorry that I'm a pain, but I was talking adding `Timestamp` inside the `Field Types` section 😬

Comment (Camvillard) [reply reference.md]: Oh, sorry, I did not get that, pushing the change rn!

**Review by gcamp — APPROVED** [review]

**Pull request merged**